avi_vantage/assets/dashboards/overview.json

```diff
@@ -0,0 +1,1543 @@
+{
+    "author_name": "Datadog",
+    "description": "",
+    "layout_type": "ordered",
+    "template_variables": [
+        {
+            "default": "*",
+            "name": "host",
+            "prefix": "host"
+        },
+        {
+            "default": "*",
+            "name": "avi_type",
+            "prefix": "avi_type"
+        },
+        {
+            "default": "*",
+            "name": "avi_tenant",
+            "prefix": "avi_tenant"
+        }
+    ],
+    "title": "Avi Vantage - Overview",
+    "widgets": [
+        {
+            "definition": {
+                "banner_img": "/static/images/integration_dashboard/avi-vantage_hero_1.png",
+                "layout_type": "ordered",
+                "show_title": false,
+                "title": "About Redis",
+                "title_align": "center",
+                "type": "group",
+                "widgets": [
+                    {
+                        "definition": {
+                            "background_color": "pink",
+                            "content": "This dashboard allows you to monitor the health of your AVI Vantage load balancers, their performance, and the traffic flowing through them.\n\n##### [Official Datadog AVI Vantage documentation \u2197](https://docs.datadoghq.com/integrations/avi_vantage/)",
```

Comment:
I'm not sure if this shows in the description (I can't access ddev.datadoghq account), but it should follow a pattern similar to the [Airflow integration](https://github.com/DataDog/integrations-core/blob/master/airflow/assets/dashboards/overview.json):
```
## Airflow\n\nThis dashboard provides a high-level overview of your Airflow instances so you can monitor metrics related to DAGs, tasks, pools and executors.\n\nFor further reading on monitoring Airflow, see:\n\n- [Official Airflow integration docs](https://docs.datadoghq.com/integrations/airflow/#data-collected)\n\nClone this template to make changes and add your own graphs and widgets.
```

Reply:
```suggestion
                            "content": "## Avi Vantage\n\nThis dashboard allows you to monitor the health of your AVI Vantage load balancers, their performance, and the traffic flowing through them.\n\nFor further reading on monitoring Avi Vantage, see:\n\n- [Official Avi Vantage integration docs](https://docs.datadoghq.com/integrations/avi_vantage/)\n\nClone this template to make changes and add your own graphs and widgets.",
```